src/server/components/auth/authenticated-user.ts

```diff
@@ -20,7 +20,7 @@ export default class AuthenticatedUser {
     return this.roles.includes(UserRoles.Administrator);
   }
 
-  async getLists() {
+  async getLists(orderByInput?: Record<string, string>) {
```

Comment:
use `orderBy?: ListsForDashboardOrderByWithRelationInput`. the type is from prisma